internal/vulnerability/runner.go

```diff
@@ -82,24 +86,12 @@ func clearTrivyCache(ctx context.Context, log *logp.Logger) error {
 	log.Info("Starting VulnerabilityRunner.ClearCache")
 	defer log.Info("Ending VulnerabilityRunner.ClearCache")
 
-	// These are the three available cli settings for clean/reset translated to flag.Options object.
-	// 	{CacheOptions: flag.CacheOptions{ClearCache: true}},
-	// 	{DBOptions: flag.DBOptions{Reset: true}},
-	// 	{MisconfOptions: flag.MisconfOptions{ResetPolicyBundle: true}},
-	// In our case we will use only the ClearCache option.
-
-	errs := make([]error, 0, 2)
-	r, err := artifact.NewRunner(ctx, flag.Options{CacheOptions: flag.CacheOptions{ClearCache: true}})
+	err := clean.Run(ctx, flag.Options{CleanOptions: flag.CleanOptions{CleanScanCache: true}})
 	if err != nil {
 		if !errors.Is(err, artifact.SkipScan) {
```

Comment:
This is no longer needed; `clean.Run` will not return `artifact.SkipScan` because it's no longer a scan command. Se we perhaps could `return clean.Run` directly.